application/templates/_helpers.tpl

```diff
@@ -68,3 +68,18 @@ reference:
   kind: Route
   name: {{ include "application.name" . }}
 {{- end }}
+
+{{- define "application.service-account-name" }}
```

Comment:
This partial template (or whatever replaces it) needs to be used in `application/templates/rolebinding.yaml` as well and anywhere else where `template "application.name" $` (+ other branches) was used to come up with a service account name.

Would be good to add test cases for the templates that use it, though that might lead to duplicated tests.

Reply:
I simplified this template a lot and used it anywhere it a service account was needed.